tests/integration/test_remoted/test_agent_communication/test_agents_switching_protocols.py

```diff
@@ -58,7 +58,6 @@ def connect_and_check_agents_status(agents, agents_connections, port, use_tcp):
         sender, injector = ag.connect(agent, protocol=TCP if use_tcp else UDP, manager_port=port)
         agents_connections[agent.id] = {'agent': agent, 'sender': sender, 'injector': injector}
         use_tcp = not use_tcp
-        assert agent.get_connection_status() == 'active'
```

Comment:
This assertion was not necessary because the `connect` method calls the `wait_status_active` method of the `agent` class. This will validate the `active` status and will make fail the test if the agent does not achieve that status.